phoenix/core/models.py

```diff
@@ -66,12 +77,13 @@ class AbstractOutage(models.Model):
     # Keeping field sales_affected for compatibility reasons. This field will also be filled with data from fields
     # lost_bookings and impact_on_turnover.
     sales_affected = models.TextField(max_length=3000, null=True, blank=True)
-    lost_bookings = models.IntegerField(null=True, blank=True)
+    lost_bookings = models.TextField(max_length=3000, null=True, blank=True)
+    lost_bookings_choice = models.CharField(
+        choices=LOST_BOOKINGS_CHOICES, max_length=5, null=False, blank=False
+    )
     impact_on_turnover = models.IntegerField(null=True, blank=True)
 
-    # Never set ETA manually. Use helper methods "set_eta" and "real_eta"
-    # for proper representation.
-    eta = models.IntegerField(null=False, blank=True, default=0)
+    eta = models.CharField(choices=ETA_CHOICES, max_length=6, null=False, blank=False)
```

Comment:
How the conversion is going to happen?  Now there are integers in the DB, but I don't see code that makes this conversion to `VARCHAR` in `migrations`

Reply:
There wasn't a need for manual conversion so I didn't think of adding one.

Reply:
The default conversion will be implicit and `1` will become `"1"`, but we have options like "<30" and all the integers that we had before will be invalid in the current field context. It could be addressed with a data migration or `USING` clause.
It would be nice to avoid invalid data in the DB right away

Reply:
I actually went with other mindset. I wanted to keep all historical data without change. As right now the ETA is any integer it seemed to me as a wrong approach to randomly change it to different values. One more smaller issue that I see with this is that the options like "<30m" could change at any point and every time that happens we would change all historical data.

To me it seems more suitable to keep historical data in previous state (when it's possible). If that's wrong approach I can make a conversion but I'd wait for the specification of all options.

Reply:
This makes sense, however, the field meaning is changed slightly - it was a certain number of minutes (I suppose), now it indicates groups of values, to which the old values belong. It is kinda has a lower precision now, which means that a single field would contain two different types of values - individual values and sets labels, which could imply additional handling logic for these cases.

Maybe one thing worries me - if we changed the way to work with eta to support the current possible options, will the old historical data be processed/displayed correctly?

We could also go in a more dynamic approach - as an idea. 
Keep the old field, but on the form level have the new values set and setup border values during the save - "<30m" => 29, etc. And to display it we could group individual values in runtime, e.g. 15 => "< 30m", 35 => "<1h", but probably it would be better to have the complete specification first.

Reply:
The values will be this:
- '<30m'
- '<2h'
- '<8h'
- '<24h'
- '>24h'

I'd prefer to make the transformation in migration file. The dynamic approach seems too implicit to me and would introduce a bit of a magic to the `ETA` field. This might cause a problem for someone who's not familiar with this.

Reply:
Ok, let's transform it separately then

Reply:
I've added the transformation